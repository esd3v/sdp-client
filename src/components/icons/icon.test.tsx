import React from 'react';
import {Icon} from './';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer
    .create(<Icon id=''/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
