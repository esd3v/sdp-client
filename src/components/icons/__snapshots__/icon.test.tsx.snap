// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.emotion-1 {
  width: 16px;
  height: 16px;
}

.emotion-0 {
  width: inherit;
  height: inherit;
}

<span
  className="emotion-1"
>
  <svg
    className="emotion-0"
  >
    <use
      xlinkHref="#"
    />
  </svg>
</span>
`;
